kyuubi-common/src/main/scala/org/apache/kyuubi/Utils.scala

```diff
@@ -147,6 +163,36 @@ object Utils extends Logging {
     dir
   }
 
+  /**
+   * write InputStream to temp file in directory
+   * @param is inputstream for file
+   * @param dir dir path for temp file creation
+   * @param fileName original file name with suffix
+   * @return created new temp file in dir in file name with random path
+   */
+  def writeToTempFile(is: InputStream, dir: Path, fileName: String): File = {
```

Comment:
please document WHO is responsible to close the `is`, caller, or this method? what will happen if things go wrong?